### DISCUSSION TITLE
pet-select fails in zsh

### DISCUSSION THREAD

**Discussion by verajosemanuel** [knqyf263/pet#281] [Q&A]
pet-select:zle:3: widgets can only be called when ZLE is active

**Comment by RamiAwar** [comment]
Can you provide more details about this error? Maybe paste your .zshrc here?

**Comment by verajosemanuel** [comment]
This was an issue from 4 years ago.

**Reply by RamiAwar** [reply]
😂

**Reply by RamiAwar** [reply]
Just trying to help! Recently joined as a maintainer and addressing all issues

**Reply by RamiAwar** [reply]
Will mark this as resolved then, thanks!
